### PR TITLE
Airflow 3: processor_poll_interval deprecated parameter usage removal

### PR DESCRIPTION
**Pull request by dirrao**
Airflow 3: processor_poll_interval deprecated parameter usage removal

### CODE REVIEW

**Comment by kaxil** [comment]
Airflow main branch is still for Airflow 2.10 :) Let's wait until the 2.10.0rc1 is created before we start removing deprecations

**Comment by dirrao** [comment]
> Airflow main branch is still for Airflow 2.10 :) Let's wait until the 2.10.0rc1 is created before we start removing deprecations

Yes.  I am tagging them with Airflow 3 to avoid accidental merging. Whenever the main branch is ready and we can start merging these MRs.

**Comment by dirrao** [comment]
@eladkal / @potiuk 
I see 2.10rc1 tag in the tags. Are we good to merge Airflow 3 changes in the main branch?

**Review by uranusjr — APPROVED** [review]

File: newsfragments/41096.significant.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed deprecated ``processor_poll_interval`` configuration parameter. Please use ``scheduler_idle_sleep_time`` configuration parameter.
```
Comment (potiuk) [inline newsfragments/41096.significant.rst]:
```suggestion
Removed deprecated ``processor_poll_interval`` configuration parameter from `scheduler` section. Please use ``scheduler_idle_sleep_time`` configuration parameter.
```

**Review by potiuk — APPROVED** [review]

**Pull request merged**